packages/ember-routing/lib/system/router.js

```diff
@@ -705,6 +706,24 @@ var EmberRouter = EmberObject.extend(Evented, {
       run.cancel(this._slowTransitionTimer);
     }
     this._slowTransitionTimer = null;
+  },
+
+  _handledErrors: computed(function() {
```

Comment:
`error[hasBeenHandledSymbol] = true`

although this wont work with primitive values, i think thats fine.